ShopifySharp/Entities/PriceBasedShippingRate.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    public class PriceBasedShippingRate : ShopifyObject
+    {
+        /// <summary>
+        /// The name of the price based shipping rate, specified by the user.
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Minimum order price
+        /// </summary>
+        [JsonProperty("min_order_subtotal")]
+        public double? MinOrderSubtotal { get; set; }
```

Comment:
Let's change the `doubles` to `decimal`, which should be more accurate and keeps the package in line with the changes made in https://github.com/nozzlegear/ShopifySharp/issues/74